builder/googlecompute/config.go

```diff
@@ -632,6 +637,18 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 		c.WindowsPasswordTimeout = 3 * time.Minute
 	}
 
+	if c.InstanceMaxRunDurationSeconds < 0 {
+		errs = packersdk.MultiErrorAppend(fmt.Errorf("'instance_max_run_duration_seconds' must be positive"))
+	}
+
+	if c.InstanceMaxRunDurationSeconds > 0 && c.InstanceTerminationAction == "" {
+		errs = packersdk.MultiErrorAppend(fmt.Errorf("'instance_termination_action' must be set if 'instance_max_run_duration_seconds' is set"))
+	}
+
+	if c.InstanceTerminationAction != "" && !(c.InstanceTerminationAction == "STOP" || c.InstanceTerminationAction == "DELETE") {
```

Comment:
It might be worth adding constants for these values.

that aside would it make sense to  defaulting to STOP after x days.